plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcClient.java

```diff
@@ -109,4 +112,23 @@ public void testMetadataWithFloatAndDoubleCol()
                 new JdbcColumnHandle("COL3", JDBC_DOUBLE, DOUBLE),
                 new JdbcColumnHandle("COL4", JDBC_REAL, REAL)));
     }
+
+    @Test
+    public void testRenameTable()
```

Comment:
Why you are testing rename table when you are changing create/drop schema? Separate commits?

Reply:
> I added a test for renameTable since it also deals with schema and table name case.

It is too vague. Please separate commits, and add dedicated tests for create and drop schemas.

Reply:
Or maybe keep it here (in this commit). It was not clear to me that you are testing interaction between create schema and create table and rename table. But please add dedicated tests for create and drop schemas.

Please also add such test to `BaseConnectorTests`, but please add there INSERT and UPDATE too (depending on `TestingConnectorBehavior`)

Reply:
I couldn't figure out a test that would only use createSchema since `getSchemaNames` lowercases all schema names before returning them. At minimum this needs to call `createTable` but I figured testing `renameTable` would give more coverage. If you're ok with that I'll move it to a separate commit.

Reply:
It was not clear for me as well. How is it related to commit message `with the correct case`?